Home/index.html

```diff
@@ -80,8 +80,11 @@ <h2>Hard Quiz</h2>
     </section>
   </main>
   
-  <footer>
+  <footer class>
     <!-- Add link to the API used -->
+    <a href="https://github.com/SchoolOfCode/w8_team-project-java-lamps">GitHub</a>
+    <a href="https://opentdb.com/api_config.php">Quiz API</a>
+    <a href="https://www.triviaplaza.com/sitemap/">Contact</a>
```

Comment:
I'm not sure about this Contact link because it doesn't send back to us if they have any questions. I would just remove it tbh.
Maybe we can add a "Website created by Java Lamps in 2021"

Reply:
Good point Clarisse, will change that now thanks